pulpcore/tests/functional/api/using_plugin/constants.py

```diff
@@ -9,13 +9,14 @@
     CONTENT_PATH
 )
 
-FILE_CONTENT_NAME = 'pulp_file.file'
 
-FILE_CONTENT_PATH = urljoin(CONTENT_PATH, 'file/files/')
+FILE_CONTENT_NAME = 'file.file'
 
-FILE_REMOTE_PATH = urljoin(BASE_REMOTE_PATH, 'file/')
+FILE_CONTENT_PATH = urljoin(CONTENT_PATH, 'files/')
```

Comment:
I'm not sure I understand this change. Shouldn't `FILE_CONTENT_PATH` still be `/pulp/api/v3/content/file/files/` and not `/pulp/api/v3/content/files/`?

Reply:
@daviddavis , you're right. Thank you!
I was surprised, why tests didn't fail, so after some investigation, I found a problem with installation of pulp_file, so test were skipped. Fixed as well now